src/domains/vote/pages/Votes/Votes.tsx

```diff
@@ -94,10 +94,7 @@ export const Votes = () => {
 	}, [activeProfile, network]);
 
 	const loadDelegates = useCallback(
-		async (wallet) => {
-			// TODO: move this to profile initialising and run it every X period
-			// eslint-disable-next-line @typescript-eslint/no-non-null-asserted-optional-chain
-			await env.coins().syncDelegates(wallet?.coinId()!, wallet?.networkId()!);
+		(wallet) => {
 			// eslint-disable-next-line @typescript-eslint/no-non-null-asserted-optional-chain
 			const delegates = env.coins().delegates(wallet?.coinId()!, wallet?.networkId()!);
```

Comment:
This should also be possible to be replaced if delegates are synced in the background.